lib/Controller/LogController.php

```diff
@@ -153,19 +162,26 @@ public function getSettings() {
 	}
 
 	/**
+	 * @AuthorizedAdminSetting(settings=OCA\LogReader\Settings\Admin)
 	 * @param bool $relative
 	 */
 	public function setRelative($relative) {
 		$this->config->setAppValue('logreader', 'relativedates', $relative);
 	}
 
 	/**
+	 * @AuthorizedAdminSetting(settings=OCA\LogReader\Settings\Admin)
 	 * @param bool $live
 	 */
 	public function setLive($live) {
 		$this->config->setAppValue('logreader', 'live', $live);
 	}
 
+	/**
+	 * @AuthorizedAdminSetting(settings=OCA\LogReader\Settings\Admin)
+	 * @param int|string $levels
+	 * @return int|string
+	 */
 	public function setLevels($levels) {
```

Comment:
```suggestion
	 */
	public function setLevels(string $levels): int {
```

Otherwise psalm complains that str_split(int ) doesn't work